### ISSUE TITLE
[SWAGGER-YAML] unexpected `example` on models

### ISSUE THREAD

**Issue by HugoMario** [swagger-api/swagger-codegen#8712]
##### Description

example fields are added on definition inside the `swagger` and `swagger-yaml` generator output

##### Swagger-codegen version

2.X

##### Swagger declaration file content or url

```yaml
paths:
  /foo:
    post:
      parameters:
        - in: body
          name: body
          schema:
            $ref: '#/definitions/BodyParam'
      responses:
        200:
          description: OK
          schema:
            $ref: '#/definitions/OKResponse'
        400:
          description: Error
          schema:
            $ref: '#/definitions/ErrorResponse'

definitions:
  BodyParam:
    type: object
    properties:
      foo:
        type: string
  OKResponse:
    type: object
    properties:
      bar:
        type: string
  ErrorResponse:
    type: object
    properties:
      error:
        type: integer
      message:
        type: string
    required:
      - error
      - message
```

##### Command line used for generation
```
java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -l swagger -i definition.yaml -o output
```
##### Suggest a fix/enhancement

This issue is happening because codegen handles operations and add examples to parameters, even if these operation are not going to be used in generator. it's just its default behave. So to fix this we just need to disable the apis generation for `swagger` and `swagger-yaml` generator, which is actually not required.

**Comment by HugoMario** [comment]
PR #8713 has been added to fix this on swagger-codegen 2.0

**Comment by HugoMario** [comment]
this has been fixed with #8713 and #8716
